### PR TITLE
Deleted file hash indicators type  - md5, sha1, sha256

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/22965#event-3152761495

## Description
Deleted file hash types from CommonServerPython under FeedIndicatorType and its usage in TAXII Server JSONFeedAPIModule and FetchIndicatorsFromFile

## Minimum version of Demisto
- [x] 5.5.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Base/Scripts/CommonServerPython/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+Deleted SHA1 type from FeedIndicatorType
+Deleted SHA256 type from FeedIndicatorType
+Deleted MD5 type from FeedIndicatorType
```
Comment (ronykoz) [inline Packs/Base/Scripts/CommonServerPython/CHANGELOG.md]: No need to add that for non-released
```suggestion
-
```

File: Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py [hunk 2]
```diff
@@ -128,18 +128,9 @@ def detect_type(indicator):
     if re.match(ipv6Regex, indicator):
         return FeedIndicatorType.IPv6
 
-    if re.match(sha256Regex, indicator):
-        return FeedIndicatorType.SHA256
```
Comment (ronykoz) [inline Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py]: should be big if with all the options and then sent as FeedIndicatorType.File

File: Packs/Base/Scripts/CommonServerPython/CHANGELOG.md (hunk 1 above)
Comment (barchen1) [reply Packs/Base/Scripts/CommonServerPython/CHANGELOG.md]: done

File: Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py (hunk 2 above)
Comment (barchen1) [reply Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py]: done

**Comment by ronykoz** [comment]
@barchen1 merge when the build finishes

File: Integrations/SymantecMSS/SymantecMSS_test.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# coding=utf-8
```
Comment (Itay4) [inline Integrations/SymantecMSS/SymantecMSS_test.py]: delete this file

Comment (barchen1) [reply Integrations/SymantecMSS/SymantecMSS_test.py]: done

**Pull request closed without merging**